content/about/faq/index.zh.md

```diff
@@ -1,6 +1,9 @@
 ---
-title: "常见问题解答"
+title: "Frequently Asked Questions"
+seo_title: "Frequently Asked Questions | Jakarta EE | The Future of Cloud Native Java"
+description: "Learn about Jakarta EE and what it means for the future of open source, cloud native enterprise Java."
```

Comment:
Others look good to me, except for this one, do we want to use English for SEO and title on Chinese page? 🤔

Reply:
I'm not sure, in my opinion, there are two types of keywords will work out on search engine in China:
 - Keywords with no difference in English and Chinese, like "Java".
 - Keywords in Chinese, like "云原生".
So maybe to the latter, use English for SEO is not a good idea

Reply:
I'm not quite sure about Baidu or other Chinese search engines, but I think Google uses meta description for SEO as well. Which is:
`description: "Learn about Jakarta EE and what it means for the future of open source, cloud native enterprise Java."` 

What do you think?  @chrisguindon 

But the title probably would be better in Chinese? Since it's showing on the page:
https://deploy-preview-1048--jakartaee.netlify.app/zh/about/faq/

Reply:
You are raising a valid concern. I suggest we create an issue to discuss improving the SEO of the Chinese translation of jakarta.ee.

We will need help for this one so let's see if the community is on board!